oag/ja-jp/Content/Topics/Access-Gateway/add-admin-app-okta-app.htm

```diff
@@ -290,30 +290,30 @@
                                                         <div class="noteOkta">
                                                             <p class="noteContent">オプションですが、Oktaはすべてのアプリケーションに証明書を含めることを推奨しています。 </p>
                                                             <p class="noteContent">証明書の一般情報については、「<a href="about-oag-certificates.htm" class="MCXref xref">証明書の用途</a>」を参照してください。証明書の取得および割り当てに関する一般的なタスクフローについては「<a href="managing-certificates-taskflow.htm" class="MCXref xref">証明書管理</a>」を参照してください。</p>
                                                         </div>
                                                         <li value="6"><span class="wintitle">［Certificates（証明書）］</span>タブを展開します。 <br /><div class="noteOkta"><p class="noteContent">デフォルトでは、アプリケーションを作成すると、システムによって自己署名付きワイルドカード証明書が生成され、アプリケーションに割り当てられます。 </p></div></li>
                                                         <li value="7"> オプション。<span class="uicontrol">[Generate self-signed certificate（自己署名付き証明書を生成）]</span>をクリックします。自己署名付き証明書が作成され、自動的にアプリケーションに割り当てられます。<br /><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="../../Resources/Images/access-gateway/cert-management-generate-self-signed.png"><img class="MCPopupThumbnail img imgthumbnail" data-mc-width="1201" data-mc-height="170" src="../../Resources/Images/access-gateway/cert-management-generate-self-signed_thumb_220_220.png" tabindex="" id="imgimg_new" /></a><br /></li>
-                                                        <li value="8">オプション。提供された証明書の一覧から既存の証明書を選択します。 <p><span class="uicontrol">[Search（検索）]</span>フィールドを使用し、共通名で証明書を絞り込みます。先へ（<span class="uicontrol">&gt;</span>）および戻る（<span class="uicontrol">&lt;</span>）矢印を使用して利用可能な証明書をナビゲートします。</p><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="../../Resources/Images/access-gateway/cert-management-select.png"><img class="MCPopupThumbnail img imgthumbnail" data-mc-width="1201" data-mc-height="431" src="../../Resources/Images/access-gateway/cert-management-select_thumb_220_220.png" tabindex="" id="imgimg_new" /></a></li>
+                                                        <li value="8">オプション。提供された証明書の一覧から既存の証明書を選択します。 <p><span class="uicontrol">検索</span>フィールドを使用し、共通名で証明書を絞り込みます。先へ（<span class="uicontrol">&gt;</span>）および戻る（<span class="uicontrol">&lt;</span>）矢印を使用して利用可能な証明書をナビゲートします。</p><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="../../Resources/Images/access-gateway/cert-management-select.png"><img class="MCPopupThumbnail img imgthumbnail" data-mc-width="1201" data-mc-height="431" src="../../Resources/Images/access-gateway/cert-management-select_thumb_220_220.png" tabindex="" id="imgimg_new" /></a></li>
```

Comment:
The global change of `Search Field（検索フィールド）` seems to have leaked from metadata to content

Reply:
@pengyao-okta FYI, maybe we can fix this in a future update

Reply:
Hi @paulwallace-okta , if I understand correctly, since we will release JA Admin UI EA, we should change all "[Search（検索）]" to "検索", right?

Reply:
We have to check for each product, we'd want the monolingual strings for products that support a localized UI -- starting next week, this will be OIE and OCE at least. 
This particular instance here is for OAG, I don't know if that UI has been localized yet?

Reply:
I will check it with global engineering team. thank you!